app/javascript/hooks/index.ts

```diff
@@ -1,2 +1,3 @@
 export { useLogger } from './use-logger'
 export { useList } from './use-list'
+export { useHighlighting } from './use-sytnax-highlighting'
```

Comment:
use-sytnax-highlighting

There's a typo here - it should be `use-syntax-highlighting` (`n` and `t` the other way around)